addons/l10n_cz/tests/test_moves.py

```diff
@@ -0,0 +1,46 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import fields
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestAccountCZ(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='cz'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.currency_usd = cls.env.ref('base.USD')
+        cls.invoice_a = cls.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'partner_id': cls.partner_a.id,
+            'date': '2024-07-10',
+            'invoice_date': '2024-07-10',
+            'currency_id': cls.currency_usd.id,
+            'company_id': cls.company_data['company'].id,
+            'invoice_line_ids': [(0, None, {
+                'name': 'test_product',
+                'product_id': cls.product_a.id,
+                'currency_id': cls.currency_usd.id,
+                'quantity': 1.0,
+                'price_unit': 1000.0,
```

Comment:
```suggestion
            'invoice_line_ids': [(0, None, {
                'quantity': 1.0,
                'price_unit': 1000.0,
```
Also, please use `Command` :nerd_face: